main/http_server/axe-os/src/app/components/swarm/swarm.component.html

```diff
@@ -62,33 +62,53 @@
 <div class="table-container">
     <table cellspacing="0" cellpadding="0" class="text-sm md:text-base">
         <tr>
-            <th>
-                <div class="flex align-items-center cursor-pointer select-none h-full" (click)="sortBy('IP')">
-                    <span class="flex-1">IP</span>
+            <th *ngFor="let field of [
+                {
+                    name: 'IP',
+                    label: 'IP'
+                },
+                {
+                    name: 'hostname',
+                    label: 'Hostname'
+                },
+                {
+                    name: 'hashRate',
+                    label: 'Hash Rate'
```

Comment:
Had to look this up, didn't know there's supposed to be a space between hash and rate.